iexec-sms-library/src/main/java/com/iexec/sms/api/SmsClient.java

```diff
@@ -80,6 +85,15 @@ String setWeb3Secret(
             String secretValue
     );
 
+    @RequestLine("POST /untee/secrets")
+    @Headers("Authorization: {authorization}")
+    SmsSecretResponse getUnTeeSecrets(
+            @Param("authorization") String authorization,
+            WorkerpoolAuthorization workerpoolAuthorization
+    );
```

Comment:
Didn't we speak about removing this endpoint ?

Reply:
Is it added for coverage in this case ?

Reply:
It was already there. I just had to reorder the methods so that it's clearer. Please see [this comment](https://github.com/iExecBlockchainComputing/iexec-sms/pull/119#discussion_r962841633) and [this commit](https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/7a1c5d207d00be368a83c878dbef9e67a83b20af).